crates/client/assets/schema.sdl

```diff
@@ -1122,10 +1134,6 @@ type VariableOutput {
 	assetId: AssetId!
 }
 
-type Version {
-	value: U8!
-}
```

Comment:
It appears that no schema types use `U8` anymore, so the SDL compiler removed it from the final SDL definition. This means I also needed to remove `U8` from the client side definitions.